### PR TITLE
Instagram posts

### PR DESCRIPTION
**Pull request by shannachau**
- Can now seed and update instagram posts
- Rspec tests are green
- Still need to test functionality in browser. Was working in console. 
- Also need to share a new INSTAGRAM_ACCESS_TOKEN. It works on Corinne's local machine, but for some reason doesn't work on mine. Was planning on asking for help tmrw when we ask about Heroku.

### CODE REVIEW

File: lib/instagram_client.rb [hunk 1]
```diff
@@ -1,8 +1,21 @@
 class InstagramClient
   INSTAGRAM_SEARCH_USERS_URI = "https://api.instagram.com/v1/users/search?client_id=#{ENV["INSTAGRAM_CLIENT_ID"]}&"
+  INSTAGRAM_FETCH_GRAMS_URI = "https://api.instagram.com/v1/users/"
+  SEED_COUNT = 5
```
Comment (catchingash) [inline lib/instagram_client.rb]: We probably want to make Instagram + Twitter match for where the seed count is located. I have no attachment to the way we had it before -- do you want to move the TwitterClient seed count stuff around?

File: app/models/post.rb [hunk 2]
```diff
@@ -46,4 +57,28 @@ def self.tweet_create_params_from_api(tweet)
     }
   end
 
+  # GRAMS ---------------------------------------------------------------------
+
+  def self.create_many_grams_from_api(grams)
+    # Instagram API returns most recent post first, but we want it in reverse
+    grams.reverse!
```
Comment (catchingash) [inline app/models/post.rb]: Why reverse them? We sort by post_time later, so it doesn't matter the db order?

File: app/models/post.rb [hunk 3]
```diff
@@ -46,4 +57,28 @@ def self.tweet_create_params_from_api(tweet)
...
+      body: gram["caption"]["text"],
+      post_time: convert_unix_to_datetime(gram["created_time"]),
+      prey_id: Prey.find_by(uid: gram["user"]["id"]).id,
+      url: gram["link"],
+      provider: "instagram"
+    }
+  end
+
+  def self.convert_unix_to_datetime(time)
+    Time.at(time.to_i).to_datetime
```
Comment (catchingash) [inline app/models/post.rb]: Do you have to do this? I thought that Rails usually handles the conversion to datetime on its own when it saves it to the db.

File: lib/instagram_client.rb (hunk 1 above)
Comment (corinnepingul) [inline lib/instagram_client.rb]: Done!

File: app/models/post.rb (hunk 2 above)
Comment (corinnepingul) [inline app/models/post.rb]: done.

**Pull request merged**